### DISCUSSION TITLE
Report on a phishing website hosted on vercel

### DISCUSSION THREAD

**Discussion by Barmanturbo** [vercel/community#6834] [Help] [answered]
### Summary

I have received an obvious phishing email, with a shortened link that lead to a vercel website.
Please take action against this user.

The website:
![image](https://github.com/vercel/community/assets/101108131/cf250e96-429d-4349-a00a-0f4539ae7442)

The link to said website:
https://security-fb-ads-accounts-1-gfc7.vercel.app/

The e-mail:
![image](https://github.com/vercel/community/assets/101108131/102cc93e-00bb-4b36-b3d9-968096b380e1)

### Steps to Reproduce

Open the link is all you need to find it.

**Comment by amyegan** [comment, marked as answer]
Thanks for reporting this. If you haven't already, please send a copy of this report to [abuse@vercel.com](mailto:abuse@vercel.com) so it can be properly tracked in our system. I'll share this post with the team when I let them know to watch for your email

**Reply by milannshrestha** [reply]
Hi @amyegan, Please update ticket #00228139

**Reply by amyegan** [reply]
I've let the team know that this discussion post is related to that case. Thanks for sharing the case number.

**Reply by Barmanturbo** [reply]
I have just sent the email with the same details.
Thank you very much!

**Reply by Barmanturbo** [reply]
The email i received returned answered with case number "00230609"

**Reply by milannshrestha** [reply]
Hi @amyegan Please follow up on following link to Discussion: 

 https://github.com/orgs/vercel/discussions/6744

**Reply by amyegan** [reply]
Thanks for sharing the case numbers.

@milannshrestha I see that your case has a response and the team has requested additional info. Please check your email for more details.

@Barmanturbo I see the case is assigned to the appropriate team and they will respond as soon as a member of the team is available to help. Please follow up with your case if you have any additional information or questions.

[a comment by an automated account is not shown]
